vifm/PKGBUILD

```diff
@@ -5,6 +5,7 @@
 pkgname=vifm
 pkgver=0.9
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `vifm`. Therefore this change should be unnecessary.